browser/ui/webui/brave_federated/federated_internals.mojom

```diff
@@ -5,12 +5,15 @@
 
 module federated_internals.mojom;
 
-struct AdStoreLog {
-    int32 log_id;
-    double log_time;
-    string log_locale;
-    int32 log_number_of_tabs;
-    bool log_label;
+struct Covariate {
+  int32 training_instance_id;
+  int32 feature_name;
```

Comment:
should it be string?

Reply:
the idea was to preserve the (integer) correspondence with the struct here, hence an integer

Reply:
I think `name` is strange naming for an integer variable. Maybe we could change it to `feature_id` or something like this.

Reply:
@LorenzoMinto if it is a type, would be better to use an enum